### PR TITLE
Jdu/match schema

### LINKED ISSUE #15: Hook up Worldreader's database with our app
Let's discuss this. I'm not sure if we want to do this now or if we should just mock it out for now.

### PR DESCRIPTION
**Pull request by johnldu**
fix #15

### CODE REVIEW

File: db/migrate/20140908072051_devise_create_users.rb [hunk 1]
```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
```
Comment (varunrau) [inline db/migrate/20140908072051_devise_create_users.rb]: Is there a reason you deleted all of these migrations?

**Comment by varunrau** [comment]
@johnldu It looks like the index name for `index_restrictedcontinent_books_on_book_id_and_geo_continent_id` is too long, so I can't run the migrations.

File: db/migrate/20140908072051_devise_create_users.rb (hunk 1 above)
Comment (johnldu) [inline db/migrate/20140908072051_devise_create_users.rb]: Basically the idea is the following:
1) delete all our old migrations
2) start our schema.rb file from their existing db schema
3) migrate forward until it matches what we had before

Comment (varunrau) [inline db/migrate/20140908072051_devise_create_users.rb]: ok that makes sense

**Comment by johnldu** [comment]
Hmm... I didn't run into that problem. Could you show me more detail?

**Comment by varunrau** [comment]
After running `rake db:reset` I get this message: 

```
ArgumentError: Index name 'index_restrictedcontinent_books_on_book_id_and_geo_continent_id' on table 'restrictedcontinent_books' is too long; the limit is 62 characters
/Users/varun/.rvm/gems/ruby-2.1.3@worldreader/gems/activerecord-4.1.5/lib/active_record/connection_adapters/abstract/schema_statements.rb:797:in `add_index_options'
```

**Comment by varunrau** [comment]
By the way did you make any changes to the `config/database.yml` file?

**Comment by johnldu** [comment]
i'll email you all the secret files.

**Comment by johnldu** [comment]
I'm not sure how to change the length issue. That index was from their schema.

**Comment by varunrau** [comment]
By the way, I think travis is failing because the database.travis.yml has not been updated

**Comment by varunrau** [comment]
Yay all the tests pass (all one of them)!

**Pull request merged**